### PR TITLE
update anomaly search backend

### PR DESCRIPTION
**Pull request by BradleySappington**
Now that we have a postgres DB, use `union` and `intersect` in standard backend query result filtering

### CODE REVIEW

**Comment by bhilbert4** [comment]
Does this mean we need to update this function as well?
https://github.com/spacetelescope/jwql/blob/develop/jwql/utils/monitor_utils.py#L225

**Comment by BradleySappington** [comment]
> Does this mean we need to update this function as well? https://github.com/spacetelescope/jwql/blob/develop/jwql/utils/monitor_utils.py#L225

Nope! That looks great

File: jwql/website/apps/jwql/data_containers.py [hunk 1]
```diff
@@ -1544,23 +1543,13 @@ def get_rootnames_from_query(parameters):
...
+                # If the rootfile info has any of the marked anomalies we want it
                 anomaly_filter = "anomalies__" + str(anomaly).lower()
-                this_anomaly_rootnames = current_ins_rootfileinfos.filter(**{anomaly_filter: True})
-                if anomaly_filters is None:
-                    anomaly_filters = this_anomaly_rootnames
-                else:
-                    anomaly_filters |= this_anomaly_rootnames
-            current_ins_rootfileinfos = anomaly_filters
-        #      >>>  END CODE PRE DB MIGRATION HERE  <<<
+                anomaly_rootfileinfos = anomaly_rootfileinfos.union(current_ins_rootfileinfos.filter(**{anomaly_filter: True}))
```
Comment (mfixstsci) [inline jwql/website/apps/jwql/data_containers.py:1551]: Much cleaner 👍

**Review by mfixstsci — COMMENTED** [review]
Thank you @BradleySappington this looks great.

**Comment by mfixstsci** [comment]
@bhilbert4 I spoke with @BradleySappington about testing the code and he performed a couple of queries before and after merging his code in to verify the results were the same. Would you mind trying the same to verify since you are more familiar with the JWST data products. Once that is done, I can go ahead and merge it in!

**Comment by bhilbert4** [comment]
@BradleySappington  @mfixstsci I just pulled this onto the dev server, restarted the service, and tried some queries. The results I'm getting back don't make any sense though. Is there something else I need to do to switch the dev server over to look at the correct database or something?

**Comment by BradleySappington** [comment]
> @BradleySappington @mfixstsci I just pulled this onto the dev server, restarted the service, and tried some queries. The results I'm getting back don't make any sense though. Is there something else I need to do to switch the dev server over to look at the correct database or something?

Comment marking that this has been resolved

**Review by bhilbert4 — APPROVED** [review]
LGTM. Spot tests on the dev server gave the expected results.

**Pull request merged**